source/versioned-api/tests/README.rst

```diff
@@ -0,0 +1,16 @@
+===================
+Versioned API Tests
+===================
+
+.. contents::
+
+----
+
+Notes
```

Comment:
We also must require that drivers test against a server running with

    --setParameter requireApiVersion=true

with auth enabled.  Otherwise, we won't be testing that drivers will work in this scenario.

Reply:
Yes, I'll add that.

While setting up the mongo-orchestration config locally I realised that this doesn't work completely yet: the server is started correctly, but the `mo` binary does not return. This is due to mongo-orchestration not sending an api version for any commands that it wants to run, thus hanging. This can't be resolved until PyMongo is updated to support the versioned API.

Reply:
Oh, ouch.  For purposes of this PR, it's sufficient to just add the prose here.  We can work out orchestration issues separately.

Reply:
Right. Added the following paragraph:
> Drivers MUST test against a server with the ``requireApiVersion`` parameter
> enabled that also requires authentication. Since API versioning options can't be
> specified using the connection string, drivers MUST rely on environment
> variables to receive the API version to use in tests. The ``apiStrict`` and
> ``apiDeprecationErrors`` options are not required for this test.

Reply:
This still leaves a small hole in test coverage, as we don't ensure that apiVersion is included for all auth mechanisms. But I think it's the best we can do right now.